packages/remark-mdx/readme.md

```diff
@@ -22,6 +22,16 @@ It’s used in [MDXjs][].
 npm install remark-mdx
 ```
 
+### Installing Other Plugins
```

Comment:
We're developing v2 which is compatible with `remark-parse@9`, so the documentation here should be clearer.

I'm a native English speaker, @ChristianMurphy  could help.